### PR TITLE
Clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10143

### CODE REVIEW

**Comment by get-itips** [comment]
I kindly request for your review @DaniEASmith

File: Teams/includes/set-up-resource-account-steps.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ For each resource account, you must assign a **Microsoft Teams Phone Resource Ac
 3. Select the resource account to which you want to assign a license. The resource account's user pane will appear.
 4. On the **Licenses and Apps** tab, under **Licenses**, select **Microsoft Teams Phone Resource Account**.
 5. Select **Save changes**.
+   
+> [!NOTE]
+> If the resource account requires a phone number, check the **Select location** drop-down in the same **Licenses and Apps** tab to match the intended country code.
```
Comment (JohanFreelancer9) [inline Teams/includes/set-up-resource-account-steps.md:70]:
```suggestion
> If the resource account requires a phone number, check that the **Select location** dropdown list in the same **Licenses and Apps** tab matches the intended country code.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by DaniEASmith — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**